api/routes/companies.js

```diff
@@ -0,0 +1,22 @@
+const router = require('express').Router({ mergeParams: true });
+const Units = require('../models/units')
+const Company = require('../models/company')
+
+// GET /api/v1/companies
+// GET ALL
+router.get('/', async (req, res, next) => {
+  const status = 200
+  const companies = await Units.find().select('company')
+  const response = await companies.find(req.query).select('_id employees name contact_email')
```

Comment:
Is this working? I would expect this to fail due to the fact that `companies` is now an array.